core/common/src/main/java/tachyon/wire/WorkerInfo.java

```diff
@@ -0,0 +1,212 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.wire;
```

Comment:
High-level comment: we should test the getters/setters, as well as the equal method.

Since all the wire structures are JSON serializable, I can think of a generic test that instantiates all the classes using reflection, serialize and serialize into another instance, and lastly compare their equality.